<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bütçe Takip Programı</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_income_expense_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</x>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Gelir/Gider Ekle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_category_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</x>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Kategori Oluştur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_analysis_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</x>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Analiz Görüntüle</string>
    </property>
   </widget>
   <widget class="QListWidget" name="categories_list">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</x>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
